### ISSUE TITLE
Question in Flickr30k dataset

### ISSUE THREAD

**Issue by hanoonaR** [jshilong/GPT4RoI#22]
Hi Authors,

I was working with the flickr30k dataset and noticed that it [returns the original bounding boxes (ori_bboxes)](https://github.com/jshilong/GPT4RoI/blob/0827109da4716d01f168bf5fa682bd0e1a874d67/gpt4roi/datasets/flickr30k.py#L312C37-L312C37) directly, whereas other referring expression datasets utilize [selected bounding boxes (select_bboxes)](https://github.com/jshilong/GPT4RoI/blob/0827109da4716d01f168bf5fa682bd0e1a874d67/gpt4roi/datasets/refcoco.py#L293C10-L294C1) to replace ori_boxes.

[`ori_bboxes = torch.cat([ori_bboxes], dim=0)`](https://github.com/jshilong/GPT4RoI/blob/0827109da4716d01f168bf5fa682bd0e1a874d67/gpt4roi/datasets/flickr30k.py#L312C37-L312C37)

Wouldn't this result in a mismatch between the region questions and the bounding boxes? Could you shed some light on this? Am I missing something?

Thank you!

**Comment by jshilong** [comment]
Thank you very much for pointing out this issue. 

I have checked it and it appears to be a problem caused by my refactoring before the release. I have fixed it. Could you please take a look at the current code to confirm if it is correct? Thank you again.
